apps/client/app/purchase-orders/[id]/page.tsx

```diff
@@ -0,0 +1,29 @@
+import { PurchaseOrder } from '../../interfaces';
```

Comment:
Wanted the initial load of the EditPurchaseOrder page to be handled by a server-side component so it is more performant.